src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,55 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import mate.academy.lib.Inject;
+
+@Entity
+@Inject
+@Table(name = "cinema Hall")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String description;
+
+    public CinemaHall() {

```

Comment:
The default constructor is not needed here if you are not going to use it. Please remove it.